app/src/main/java/com/example/androidseminar/ui/RepoListAdapter.kt

```diff
@@ -0,0 +1,38 @@
+package com.example.androidseminar.ui
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.androidseminar.data.RepoInfo
+import com.example.androidseminar.databinding.ItemRepoBinding
+
+class RepoListAdapter : RecyclerView.Adapter<RepoListAdapter.RepoViewHolder> () {
+
+    val repoList = mutableListOf<RepoInfo>()
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): RepoViewHolder {
+        val binding = ItemRepoBinding.inflate(
+            LayoutInflater.from(parent.context),
+            parent,
+            false
+        )
+        return RepoViewHolder(binding)
+    }
+
+    override fun getItemCount(): Int = repoList.size
+
+    override fun onBindViewHolder(holder: RepoViewHolder, position: Int) {
+        holder.onBind(repoList[position])
+    }
+
+    class RepoViewHolder(
+            private val binding: ItemRepoBinding
+    ) : RecyclerView.ViewHolder(binding.root) {
+        fun onBind(repoInfo: RepoInfo) {
+            binding.tvRepoName.text = repoInfo.repoName
+            binding.tvRepoDisc.text = repoInfo.repoDisc
+            binding.tvRepoLanguage.text = repoInfo.repoLanguage
```

Comment:
이부분은 진희한테도 코멘트 달았던 부분인데 지금 사용하기는 어려워도 한번 들어보고 한번 찾아보면 좋을 것 같아서 달아봐 ㅎㅎ
나중에 item_repo 에서 <data></data> 안에 RepositoryInfo형 변수를 선언한다면,
예를들어 변수명을 repoData로 선언했다고치면
```suggestion
            binding.repoData = repoInfo
```
이렇게 간단하게도 처리가 가능할거야! 
저렇게 처리가 되면 
tvRepoName에서  `android:text = "@{repoData.repoName}" ` 이런식으로 사용이 가능해!

위의 방법이 어떤건지는 한번 찾아보는 것도 좋을 것 같아!!  (ps. 리사이클러뷰 데이터바인딩으로 검색해도 많이 나올거야)

Reply:
오 고마워! 한 번 적용해볼게!

Reply:
맞아 연주 말대로 저렇게 처리하면 좋아유! 나도 이렇게 얼른 리팩토링 해야겠다..